components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -0,0 +1,47 @@
+# Adding new metrics
+
+All metrics that your application collects must be defined in a `metrics.yaml`
+file. This file should be at the root of the application module (the same
+directory as the `build.gradle` file you updated). The format of that file is
+documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
```

Comment:
Linking to it makes sense for now, but eventually we'll probably want to bring that over here as part of the [reorg documentation bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1529761).